### ISSUE TITLE
Translation to Portuguese

### ISSUE THREAD

**Issue by clente** [hadley/r-pkgs#772]
I just saw that community translations are now allowed for R4DS: https://github.com/hadley/r4ds/issues/955.

Does this also apply to _R Packages_?

**Comment by hadley** [comment]
Yes, we'll definitely negotiate this in the contract for the 2e.

**Comment by clente** [comment]
Thanks! Please let me know when we can start working on it! 🎉

**Comment by jennybc** [comment]
You'll have to wait until July or so, for the content to be stable enough to begin a translation effort.

**Comment by clente** [comment]
Thanks a lot @jennybc. Do you need help with the content? I see there are a lot of open issues, but I'm not sure if they are intended for the general public.

**Comment by jennybc** [comment]
Not at this moment, no. I'll be working through the un-revised chapters holistically, one by one.

**Comment by brunomioto** [comment]
Hi there!
Is the book good to start the translation? 
Now that the R4DS (PT-BR) translation is done, I can help @clente with it!
